client/notifications/src/panel/boot/stylesheets/shared/colors.scss

```diff
@@ -1,3 +1,3 @@
 $wpnc__yellow-dark:    darken( $alert-yellow, 5% ); // #f0b849
-$wpnc__red-darker:     darken( $alert-red, 32% ); // #6d1818
+$wpnc__red-darker:     var( --color-error-800 ); // #6d1818
```

Comment:
hmmmm... given the troubles we saw with missing var defs yesterday, this may break the standalone notifications build?

Reply:
good catch; let's keep them for now, I'll look at them separately 👍

Reply:
that probably also counts for https://github.com/Automattic/wp-calypso/pull/30458/files#diff-34868b206dce1a4863158a2cb4922636 ?

Reply:
yeah, any color variables in there. Ideally we figure out how to turn off preserve for sdk builds and keep the custom props.

Reply:
> Ideally we figure out how to turn off preserve for sdk builds and keep the custom props.

As that is now merged in #30503 we should be good here, right?

Reply:
all good!